roles/confluent.common/tasks/rbac_setup.yml

```diff
@@ -18,8 +18,6 @@
     kafka_cluster_id: "{{ cluster_id_query.json.data[0].cluster_id }}"
   when: (cluster_id_source | default('erp') == 'erp') and not ansible_check_mode
 
-
-# TODO this will not work for Secure ZK or Archive installation method
```

Comment:
Is this no longer the case?

Reply:
The hardcoded path and port has been parameterised in a different PR. The original branch of this PR didn't have this comment which has been removed now. 
So, this task runs with secure zookeeper and archive location now.